chrome/content/about.xhtml

```diff
@@ -188,11 +189,12 @@
    <p>
      Nostalgy comes with a backup system for rules. In the Preferences
      dialog (tab "Rules"), there is a button "Send rules by email".
+     (Previous versions only, not in TB 68)
```

Comment:
I suspect the new version will not support older TB, so I suggest to adapt the documentation and simply remove the things which are not longer available in the new version.

Reply:
ok